### DISCUSSION TITLE
neue Funktionen: (erste) Daten löschen und/oder Flag "Daten nutzbar"?

### DISCUSSION THREAD

**Discussion by jentie** [technologiestiftung/stadtpuls#10] [Ideas]
Die einfache Oberfläche finde ich sehr gelungen, allerdings fehlt mir aus der Praxis eine Funktion: Beim einrichten eines neuen Sensors (ggf. auch noch während der Anbringung und Inbetriebnahme) fallen erste Daten an, die man zur Kontrolle sehen möchte, die aber noch nicht gültig sind oder nicht die letztlich geplante Qualität haben. 
Eine einfache Funktion wäre, alle bisherigen Daten löschen zu können. Dann könnte man den eigenen Sensor in Betrieb nehmen, ggf. auch ein paar Tage testweise laufen lassen und dann mit dem löschen der bisherigen Daten sauber beginnen. Alternativ oder zusätzlich könnte man auch ein Flag setzen, dass den Nutzenden anzeigt, dass die Daten als stabil und produktiv nutzbar angesehen werden (oder andersrum, dass man noch in der Installationsphase ist).  
Jens Tiemann

**Comment by ff6347** [comment]
Hey Jens. Danke für deinen Input. Das editieren der Daten für euch Maker steht schon in unserem backlog. 

Bis wir einen Editor implementiert haben, muss das leider noch manuell geschehen. Richte doch erstmal deinen Sensor ein. Wenn wir Test Daten entfernen sollen nennst du uns den Zeitraum und wir machen das für dich.

**Reply by jentie** [reply]
Hi Fabian, danke für die Info. Mir gefällt die einfache Oberfläche, insofern ist die genannte Funktion vielleicht auch was für den geplanten Editor? Irgendwo (Adafruit IO?) musste / habe ich jede Menge Messpunkte einzeln gelöscht, dann lieber auf ein paar gute Messungen verzichten, zumal die Sensoren ja über lange Zeiträume laufen sollen, da kann man dann eher mal einen Tag komplett löschen. Das ist natürlich erstmal nur (m)eine Sichtweise...

**Comment by ff6347** [comment]
Batch Operationen wären schon besser als Flags.

Flags würde ich ungerne einführen, da das bedeutet jedem Record was mitzugeben oder aber dem Sensor ein flag zu geben und dann immer dagegen eine Query laufen zu lassen. Somit müsste das Flag einen Zeitraum mit sich tragen, bzw mehrere Zeiträume, da User das Flag ja setzen und wieder raus nehmen könnten usw. das hat auf jeden Fall eine höhere Komplexität als ein Editor. IMO

**Reply by jentie** [reply]
ja, stimmt, daran habe ich nicht gedacht
